api/client.yaml

```diff
@@ -1589,6 +1831,33 @@ components:
           description: Customers name suffix. "Jr", "PH.D."
         type:
           $ref: '#/components/schemas/CustomerType'
+        businessName:
+          type: string
+          description: Business Name for business type customers
+          example: Alphabet Inc.
+        doingBusinessAs:
+          type: string
+          description: Doing Business As (DBA) name for business type customers
+          example: Google
+        businessType:
+          type: string
+          description: Business Type for business type customers - "Individual, sole proprietor, or single-member LLC", "Corporation", "Nonprofit organization", "Partnership", "Limited Liability Company"
+        ein:
+          type: string
+          description: Employer Identification Number (EIN) for business type customers
+          example: 77-0493581
+        duns:
+          type: string
+          description: Dun & Bradstreet D-U-N-S Number (D-U-N-S) for business type customers
+          example: 15-048-3782
+        sicCode:
+          type: string
+          description: SIC Code (https://docs.google.com/spreadsheets/d/1erIdqoy60JwLAnpb91EfoJV5YrXDnbwSaA-aqcBlw48/edit#gid=1627409140)
```

Comment:
Where is this doc from? Why not embed the values into our OpenAPI doc?

Reply:
This doc was from Notion, in the description for this epic.  Since there are 85 different options, I felt like there were too many options to list here

Reply:
We'd want the enum generated though, right? That way our code can include them? 

Perhaps we should include the SIC codes in a different package to validate with?

Reply:
I can add it as an emum along with the NAICS codes

Reply:
This should be resolved with my latest commits